### ISSUE TITLE
[High Contrast]: Button: style is different from expected.

### ISSUE THREAD

**Issue by bharay** [OfficeDev/office-ui-fabric-react#1441]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: edge, Windows 10.

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes! but I will have to learn how HC styles work)

Requested priority: (Normal) 

#### Describe the issue:

HC button style is different from expected style.

#### Actual behavior:
_Green text on black background with no border_
![image](https://cloud.githubusercontent.com/assets/5851349/24798819/b807e122-1bb5-11e7-84a3-0c14a60680f2.png)
#### Expected behavior:
_White text on black background with white border_
![image](https://cloud.githubusercontent.com/assets/5851349/24798831/c84f7e46-1bb5-11e7-96c0-06644dfadb91.png)

**Comment by micahgodbolt** [comment]
@bharay Check out this bit of code https://github.com/OfficeDev/office-ui-fabric-react/blob/39deee70abb7fd60485df84a3b5a1c05a98871c4/packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.scss#L16

High contrast is a media query, so you can change the border color only when high contrast is active.

```
 @media screen and (-ms-high-contrast: active) {
    border: $size solid $active-color;
  }
```

**Comment by micahgodbolt** [comment]
We're having a big bug bash on Tuesday, so we'll probably knock this out then. But if you get a chance to look at it before then, feel free to submit a PR. We'll be happy to review.

**Comment by betrue-final-final** [comment]
This is partially by design. That flavor of button is a command bar button, and not stand alone. It should NOT have a border in the rest state, but should be white.

**Comment by aleksandrjPersonal** [comment]
Add borders to all buttons except Icon button and CommandBar button. 1px solid white. per @betrue-final-final
